tests/test_kerascv.py

```diff
@@ -2,7 +2,7 @@
 
 import json
 import keras_cv
-import keras
```

Comment:
I think we should have separate tests for `keras_core` to ensure we still test regular `keras`. 

Also, once `keras_core` is GA, it will simply become the `keras==3.0` so we will be able to simply delete the new `test_keras_core.py`.